Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/bugreporting/storage/repository/BugRepository.kt

```diff
@@ -0,0 +1,11 @@
+package de.rki.coronawarnapp.bugreporting.storage.repository
+
+import androidx.lifecycle.LiveData
+import de.rki.coronawarnapp.bugreporting.event.BugEvent
+
+interface BugRepository {
+    fun getAll(): LiveData<List<BugEvent>>
+    fun get(id: Long): LiveData<BugEvent>
```

Comment:
why are those life data?

Reply:
IMHO it would be good to align these function names with the ones in BugDao

Reply:
@chris-cwa LiveData is removed now.